pkg/archiver/walker.go

```diff
@@ -55,11 +55,18 @@ func Walker(ctx context.Context, rootPath string, options ...WalkerOption) (<-ch
 		o(&opts)
 	}
 
-	_, err := os.Stat(rootPath)
+	info, err := os.Lstat(rootPath)
 	if err != nil {
 		return nil, err
 	}
 
+	if info.Mode()&os.ModeSymlink == os.ModeSymlink {
+		rootPath, err = filepath.EvalSymlinks(rootPath)
```

Comment:
as a suggestion, should we just use `os.Readlink` (https://godoc.org/os#Readlink) here, as it seems to be closer to what we're looking for? `EvalSymlinks` should do more than just final destination?

Reply:
sorry, was my misunderstanding